### PR TITLE
Add README.md to guidelines folder

### PR DESCRIPTION
**Pull request by MaxWofford**
This is associated with #396

### CODE REVIEW

File: guidelines/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Guidelines
+
+The objective of guidelines is to share knowledge between club leaders in a more
```
Comment (jonleung) [inline guidelines/README.md]: I don't understand what this paragraph is trying to tell me. @MaxWofford could you please try to reword? Happy to talk about it as well.

**Comment by jonleung** [comment]
Left a comment.

Also, @MaxWofford, in the future, it would be really helpful if you put pull requests you assign to me in the `Review` ZenHub column.

Thanks!

**Comment by MaxWofford** [comment]
> it would be really helpful if you put pull requests you assign to me in the `Review` ZenHub column

@jonleung Will do!

File: guidelines/README.md (hunk 1 above)
Comment (MaxWofford) [inline guidelines/README.md]: @jonleung What part is confusing?

**Comment by jonleung** [comment]
@MaxWofford sorry it took me so long to merge this, just let me know when the branch is in sync again.

**Comment by MaxWofford** [comment]
@jonleung You can do that.
screenshot from 2015-10-28 15 23 00

**Comment by MaxWofford** [comment]
@jonleung Also your comment was [never addressed](https://github.com/hackedu/hackedu/pull/397#discussion_r42949090)

**Comment by MaxWofford** [comment]
@jonleung Assigning back to you for responses to comments.

File: guidelines/README.md (hunk 1 above)
Comment (jonleung) [inline guidelines/README.md]: I guess I don't understand is the usefulness of this readme.

Comment (MaxWofford) [inline guidelines/README.md]: It differentiates what to put in workshops and what to put in guidelines. Is that unclear from the way it is worded?

**Comment by MaxWofford** [comment]
@paked Are you available to review this?

**Comment by paked** [comment]
@MaxWofford on it.

File: guidelines/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+collaborative medium than case_studies. Technical guides and lessons should be
```
Comment (paked) [inline guidelines/README.md]: I don't think `Technical guides` is the right word to use here

File: guidelines/README.md (hunk 1 above)
Comment (paked) [inline guidelines/README.md]: I wouldn't understand this either, or at least wouldn't be able to contribute to it...

File: guidelines/README.md (hunk 2 above)
Comment (MaxWofford) [inline guidelines/README.md]: @paked Perhaps `curricular`?

File: guidelines/README.md (hunk 1 above)
Comment (MaxWofford) [inline guidelines/README.md]: @paked Thanks for the input! I'll close the PR for now and we can reopen it if anyone ever has questions about what to put in the `/guidelines` folder.

**Pull request closed without merging**